src/AngleSharp.Css/Values/Primitives/Constant2.cs

```diff
@@ -0,0 +1,80 @@
+namespace AngleSharp.Css.Values
+{
+    using System;
+
+    /// <summary>
+    /// Represents a selected CSS enum value.
+    /// </summary>
+    struct Constant<T,U> : ICssPrimitiveValue, IEquatable<Constant<T,U>>
```

Comment:
Is this really used? I only see one line with 2 types and that is not really used (see question).